src/Controller/TalkCommentsController.php

```diff
@@ -189,4 +194,13 @@ public function updateComment(Request $request, PDO $db)
 
         return $commentMapper->getCommentById($commentId);
     }
+
+    private function getCommentMapper(Request $request, PDO $db): TalkCommentMapper
+    {
+        if ($this->commentMapper === null) {
+            return new TalkCommentMapper($db, $request);
```

Comment:
Perhaps you meant this?

```suggestion
            $this->commentMapper = new TalkCommentMapper($db, $request);
```